include/deal.II/grid/grid_tools_cache.h

```diff
@@ -72,18 +72,19 @@ namespace GridTools
      * If you provide the optional `mapping` argument, then this is used
      * whenever a mapping is required.
      *
-     * @param tria The triangulation for which to store information
+     * @param tria The triangulation for which to store information.
      * @param mapping The mapping to use when computing cached objects
```

Comment:
Since you add the period in the other line, do you want to add it here as well?
```suggestion
     * @param mapping The mapping to use when computing cached objects.
```